pkg/build/gobuild.go

```diff
@@ -331,6 +331,8 @@ func getDelve(ctx context.Context, platform v1.Platform) (string, error) {
 	// install delve to tmp directory
 	args := []string{
 		"build",
+		"-trimpath",
```

Comment:
- `trimpath`:
  - [Perfectly Reproducible, Verified Go Toolchains](https://go.dev/blog/rebuild)
    - For Go programs that don’t need cgo, a reproducible build is as simple as compiling with CGO_ENABLED=0 go build -trimpath. Disabling cgo removes the host C toolchain as a relevant input, and `-trimpath removes the current directory`.

- `"-ldflags=-s -w"`
  - [[Go: Reproducible Builds](https://crashonly.com/)]([Go: Reproducible Builds](https://crashonly.com/))
    - `-ldflags: reduce binary size by removing debug tables`
    - `-trimpath: remove paths from stacktraces`